File: app/src/main/java/net/rmoreno/weatherapp/ui/WeatherView.java
package net.rmoreno.weatherapp.ui;

import net.rmoreno.weatherapp.models.CurrentWeather;
import net.rmoreno.weatherapp.models.DailyWeather;

import java.util.ArrayList;

public interface WeatherView {

    void displayDailyWeather(ArrayList<DailyWeather> dailyWeather, int sweaterTemp);

    void displayCurrentWeather(CurrentWeather currentWeather);
}

File: app/src/main/java/net/rmoreno/weatherapp/ui/WeatherView.kt
package net.rmoreno.weatherapp.ui

import net.rmoreno.weatherapp.models.Currently
import net.rmoreno.weatherapp.models.DailyDetail

interface WeatherView {
    fun displayDailyWeather(dailyWeather: List<DailyDetail>, sweaterTemp: Int, timezone: String)

    fun displayCurrentWeather(currentWeather: Currently, timezone: String)

    fun setLoading(isLoading: Boolean)

    fun displayNetworkError()

    fun goToIntroActivity()

}
